### DISCUSSION TITLE
Welcome to CodeSnap Discussions!

### DISCUSSION THREAD

**Discussion by RanitManik** [RanitManik/CodeSnap#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gargvishu2000** [comment]
Hi, i think it is not working when taking input 
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner scn=new Scanner(System.in);
        System.out.println("Enter the value of n");
        int n=scn.nextInt();
        System.out.println("Hello, World!");
        System.out.println(n);
        }
    }

**Reply by RanitManik** [reply]
In fact, we are currently utilizing the Piston API and do not support command-line inputs. Therefore, this approach will not be feasible. In the future, we plan to host our own code runner. We apologize for any inconvenience this may cause.
